shared/css/conversation.css

```diff
@@ -967,6 +967,31 @@ html[dir="rtl"] .text-chat-entry.received > p:after {
   align-self: auto;
 }
 
+.text-chat-notif {
+  display: flex;
+}
+
+.text-chat-notif > .content-wrapper {
+  margin-left: 10px;
+  margin-bottom: 10px;
+  display: flex;
+  border: 1px solid #B1B1B1;
+  max-width: 70%;
+}
+
+.text-chat-notif img {
```

Comment:
Please use child selectors for these, they have better performance even if you have to be a bit more explicit.